features/steps/browser_steps.rb

```diff
@@ -31,6 +31,9 @@
   if err['columnNumber']
     step("the \"columnNumber\" of stack frame 0 equals #{err['columnNumber']}")
   end
+  if err['file']
+    step("the \"file\" of stack frame 0 ends with \"#{err['file']}\"")
```

Comment:
this is an "ends with" match so that the hard coded values don't need to contain the hostname/port/protocol